src/js/Text/index.ts

```diff
@@ -0,0 +1,57 @@
+import Control, {ControlProps} from '../Control';
+import { element, any } from 'prop-types';
+
+type TextProps = ControlProps & {
+  value: string;
```

Comment:
Please, change 'value' to 'value?', because this option is not required. Also, please update default values of constructor on User Guide.
Thank you!